web/src/mobile/translations/en.ts

```diff
@@ -30,6 +30,9 @@ export const en = {
     'page.mobile.orders.header.orderType.buy.limit': 'Buy limit',
     'page.mobile.orders.header.orderType.sell.market': 'Sell market',
     'page.mobile.orders.header.orderType.sell.limit': 'Sell limit',
+    'page.mobile.orders.header.orderType.buy.stop_loss': 'Stop loss',
+    'page.mobile.orders.header.orderType.sell.take_profit': 'Take profit',
+    'page.mobile.orders.header.orderType.buy.stop_limit': 'Stop limit',
```

Comment:
please apply same translations for mobile as we have for desktop:
```
    'page.mobile.orders.header.orderType.stop_loss': 'Stop-loss',
    'page.mobile.orders.header.orderType.stop_limit': 'Stop-limit',
    'page.mobile.orders.header.orderType.take_limit': 'Take-limit',
    'page.mobile.orders.header.orderType.take_profit': 'Take-profit',
```

Reply:
also, please notice that currently translation string is build with orderSide [buy or sell] and orderType. Now we need to build translation string only with orderType.

If order item has side 'buy' we put green color for **Filled** and **Order type** data.
Example you can find in Orders table for desktop: https://github.com/openware/baseapp/blob/637796d9d2002d3fef5d980c2ed51ad72d4e7b1f/web/src/containers/OpenOrders/index.tsx#L86